colorbleed/plugins/maya/publish/collect_renderlayers.py

```diff
@@ -135,4 +138,10 @@ def parse_options(self, render_globals):
         state = "Suspended" if attributes["suspendPublishJob"] else "Active"
         options["suspendPublishJob"] = state
 
+        # Check if the run slap comp
+        if attributes["runSlapComp"]:
```

Comment:
This will break backwards compatibility for render global nodes that don't have the slap comp attribute yet, right?